src/videotile/DefaultVideoTile.ts

```diff
@@ -97,9 +97,12 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
 
       // Need to yield the message loop before clearing `srcObject` to
       // prevent Safari from crashing.
+      const prevSrcObject = videoElement.srcObject;
       if (new DefaultBrowserBehavior().requiresVideoElementWorkaround()) {
         AsyncScheduler.nextTick(() => {
-          videoElement.srcObject = null;
+          if (videoElement.srcObject === prevSrcObject) {
```

Comment:
When would this not be true? On like 100, arent you setting perSrcObject to videoElement.srcObject?

Reply:
So the issue is when this code is executed, the `srcObject` in the video element may already change. So I store a reference to the previous `srcObject` to make sure we do not clear out a new one.

Reply:
Should we move that declaration of `prevSrcObject` inside the workaround condition since this applies only to the workaround or is that used somewhere else as well?